ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -110,6 +117,13 @@ def check(self, _):
             # Always close the client, failure to do so leaks one thread per connection left open
             client.close()
 
+    def initialize_client(self):
+        """
+        # TODO: this indirection is useful for swapping out client, for example for testing
+        """
```

Comment:
```suggestion
        # TODO: this indirection is useful for swapping out client, for example for testing
```

Reply:
@dkirov-dd thanks for proactively jumping in and calling out that todo. This one isn't quite ready yet, I'm checking if customer would like to try this out before we commit to it. I'll be sure to address the todo before I submit it for review, yell at me if I don't ;)